### PR TITLE
store/tikv: turn on Decimal, Avg, Sum switch and fix `First` function

### PR DESCRIPTION
**Pull request by coocood**
Also fix a bug about evaluating `first` aggregation function, that `first` should be the first not null value rather than first value.

### CODE REVIEW

**Comment by coocood** [comment]
@shenli @BusyJay @zimulala PTAL

**Comment by hanfei1991** [comment]
LGTM

File: store/tikv/mock-tikv/aggregate.go [hunk 1]
```diff
@@ -202,7 +202,7 @@ func (n *aggregateFuncExpr) updateCount(ctx *selectContext, args []types.Datum)
 
 func (n *aggregateFuncExpr) updateFirst(ctx *selectContext, args []types.Datum) error {
 	aggItem := n.getAggItem()
-	if aggItem.evaluated {
+	if aggItem.evaluated && !aggItem.value.IsNull() {
```
Comment (shenli) [inline store/tikv/mock-tikv/aggregate.go]: aggItem.evaluated  is useless now.

**Comment by zimulala** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**